store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/TableUtils.java

```diff
@@ -216,7 +221,63 @@ public static Connector getConnector(final String instanceName, final String zoo
         try {
             return instance.getConnector(userName, new PasswordToken(password));
         } catch (final AccumuloException | AccumuloSecurityException e) {
```

Comment:
make this final Exception

Reply:
This was already like this. It could be changed in the same way as the others, but this PR is not the place to do it as the code hasn't been introduced or modified. There are another 5 instances in this class where the least upper bound (most specific supertype of the types in question) of a multi-catch clause is Exception. If we want to replace these (not worth doing) then it should come under a separate code clean up issue.